### PR TITLE
Fix NXP-30469 video conversion timeout master

### PR DESCRIPTION
**Pull request by bdelbosc**

### CODE REVIEW

**Comment by efge** [comment]
Second commit has incorrect ticket number NXP-30070.

File: modules/platform/nuxeo-platform-video/src/main/java/org/nuxeo/ecm/platform/video/VideoHelper.java [hunk 1]
```diff
@@ -255,4 +266,26 @@ public static VideoInfo getVideoInfo(Blob video) {
...
+     * Commits and starts a new transaction with a custom timeout.
+     *
+     * @since 11.5
+     */
+    public static void newTransaction() {
+        if (TransactionHelper.isTransactionActiveOrMarkedRollback()) {
+            TransactionHelper.commitOrRollbackTransaction();
+        }
+        if (log.isDebugEnabled()) {
+            log.debug("Commit and start transaction with timeout " + getTransactionTimeout() + "s");
```
Comment (troger) [inline modules/platform/nuxeo-platform-video/src/main/java/org/nuxeo/ecm/platform/video/VideoHelper.java:279]: Worth extracting `getTransactionTimeout()` to a var?

Comment (bdelbosc) [reply modules/platform/nuxeo-platform-video/src/main/java/org/nuxeo/ecm/platform/video/VideoHelper.java:279]: not necessary for a debug trace

**Review by kevinleturc — APPROVED** [review]

File: modules/platform/nuxeo-platform-video/src/main/java/org/nuxeo/ecm/platform/video/VideoHelper.java [hunk 2]
```diff
@@ -255,4 +266,26 @@ public static VideoInfo getVideoInfo(Blob video) {
...
+        }
+        TransactionHelper.startTransaction(getTransactionTimeout());
+        // timeout of command line executions will be aligned with the transaction timeout
+    }
+
+    public static int getTransactionTimeout() {
```
Comment (ataillefer) [inline modules/platform/nuxeo-platform-video/src/main/java/org/nuxeo/ecm/platform/video/VideoHelper.java:285]: Add `@since 11.5`, or make it protected?

**Review by ataillefer — DISMISSED** [review]

**Review by kevinleturc — APPROVED** [review]

**Review by troger — DISMISSED** [review]

**Review by ataillefer — DISMISSED** [review]

**Review by ataillefer — APPROVED** [review]

**Review by troger — APPROVED** [review]

**Pull request merged**